### PR TITLE
Adds a framework for radio language translation

### PR DESCRIPTION
**Pull request by Cyberboss**
No gameplay changes yet

### CODE REVIEW

**Comment by MoreRobustThanYou** [comment]
Maybe we can bring NTSL back...? *flashback to the cool NTSL scripts on paradise which made a colored box with the job in it, and the annoying scripts that auto translated the 10 different languages*

**Comment by Fox-McCloud** [comment]
>flashback to the cool NTSL scripts on paradise which made a colored box with the job in it, and the annoying scripts that auto translated the 10 different languages

NTSL is literal cancer.

File: code/game/machinery/telecomms/broadcasting.dm [hunk 1]
```diff
@@ -86,61 +86,61 @@
...
-
-	for(var/mob/R in receive) //Filter receiver list.
-		if (R.client && R.client.holder && !(R.client.prefs.chat_toggles & CHAT_RADIO)) //Adminning with 80 people on can be fun when you're trying to talk and all you can hear is radios.
-			receive -= R
-
-	for(var/mob/M in GLOB.player_list)
-		if(isobserver(M) && M.client && (M.client.prefs.chat_toggles & CHAT_GHOSTRADIO))
-			receive |= M
+		if(freqtext in GLOB.reverseradiochannels)
+			for(var/I in GLOB.all_radios["[GLOB.SYND_FREQ]"])//syndicate radios use magic that allows them to hear everything. this was already the case, now it just doesn't need the allinone anymore. solves annoying bugs that aren't worth solving.
```
Comment (optimumtact) [inline code/game/machinery/telecomms/broadcasting.dm]: seems like a hack

**Review by optimumtact — APPROVED** [review]

File: code/game/machinery/telecomms/broadcasting.dm (hunk 1 above)
Comment (Cyberboss) [inline code/game/machinery/telecomms/broadcasting.dm]: trying to touch as little saycode as possible

File: code/game/machinery/telecomms/broadcasting.dm [hunk 2]
```diff
@@ -86,61 +86,61 @@
...
+				radios |= I
 
 	var/rendered = virt.compose_message(virt, language, message, freq, spans) //Always call this on the virtualspeaker to advoid issues.
-	for(var/atom/movable/hearer in receive)
-		hearer.Hear(rendered, virt, language, message, freq, spans)
 
-	if(length(receive))
+	var/list/received = 0
```
Comment (lzimann) [inline code/game/machinery/telecomms/broadcasting.dm]: why is this `list` being assigned a 0

**Pull request closed without merging**